### DISCUSSION TITLE
USB-Hub 的接口数量

### DISCUSSION THREAD

**Discussion by rootkiter** [peng-zhihui/HelloWord-Keyboard#56] [Ideas]
USBHUB 有点少，两个 USB-A以外,再加两个USB-C的扩展，应该能更好。
供电问题其实可以设计双供电口。日常一个，加负载的时候，用第二个供电口辅助供电。这样也能避免电池老化的问题。

**Comment by tklovejk** [comment]
那得加usbhub芯片
